xcarchive/xcarchive.go

```diff
@@ -39,3 +41,30 @@ func IsMacOS(archPath string) (bool, error) {
 
 	return exist, nil
 }
+
+// UnzipXcarchive ...
+func UnzipXcarchive(xcarchivePth string) (string, error) {
+	tmpDir, err := pathutil.NormalizedOSTempDirPath("__xcarhive__")
+	if err != nil {
+		return "", err
+	}
+
+	if err := ziputil.UnZip(xcarchivePth, tmpDir); err != nil {
```

Comment:
please move to the end of the block, instead of returning nil just
return tmpDir, ziputil.UnZip(xcarchivePth, tmpDir)

Reply:
Fixed.